### DISCUSSION TITLE
Install on plex fails.

### DISCUSSION THREAD

**Discussion by Ratingthomas** [kimai/kimai#4350] [On-Premise: Installation & Updates]
### Describe the issue

I installed Kimai following [the plesk](https://www.kimai.org/documentation/installation.html#plesk) guide, with terminal access. But when access the site, my css doesn't load. I have my permissions correct.

Mac safari 16

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

2.0.35

### How do you run Kimai?

Plesk

### Which PHP version are you using?

8.2

### Logfile

Not found

### Screenshots

<img width="936" alt="SCR-20231017-odze" src="https://github.com/kimai/kimai/assets/92450283/0c03c002-6090-4047-8c4e-0e1bccf60c38">
<img width="960" alt="SCR-20231017-ofwp" src="https://github.com/kimai/kimai/assets/92450283/06e82de3-590a-45fe-b7ad-2f89eb5154c7">

**Comment by vida10-buqueinsignia** [comment]
I am facing the same issue. There is any fix for it at the moment?
